### ISSUE TITLE
Sync Timers Across Multiple PCs

### ISSUE THREAD

**Issue by mikereader1** [runelite/runelite#5395]
I, probably like many others, play on multiple computers for various reasons. For example, I may be watching TV AFK playing on my laptop or hardcore PVMing/PKing on my desktop. I usually hop between multiple computers in a single day. 

It would be cool if the Account feature also sync all of the timers. I'm not quite sure how possible this is, but it would help me out tremendously in all my farming and birdhouse runs.  I'm currently working towards 99 Farming and I use nearly every available farming patch in the game. 

TL:DR
Simple put, it is hard to keep track of all my timers while moving from PC to PC and this would be a huge quality of life improvement for me.

**Comment by deathbeam** [comment]
Just login to RuneLite, it is already a feature

**Comment by mikereader1** [comment]
wow, I'm an idiot...my family member logged into a separate runelite account on my laptop. 

facepalm. Ty deathbeam you are truly a god of the RS community.
